packages/calypso-e2e/src/lib/flows/start-site-flow.ts

```diff
@@ -97,7 +97,7 @@ export class StartSiteFlow {
 	 * @param {string} vertical Name of the vertical to select
 	 */
 	async enterVertical( vertical: string ): Promise< void > {
-		await this.page.waitForLoadState( 'networkidle', { timeout: 20 * 1000 } );
+		await this.page.waitForLoadState( 'networkidle', { timeout: 30 * 1000 } );
```

Comment:
This change has fixed the problem, but not entirely, this test looks flaky. Perhaps we can check it further on how we can improve it without this timeout.

Reply:
Feel free to open an issue @renancarvalho! This PR is for addressing this quickly as there are people blocked in their PRs 👍

Reply:
I'm going to agree that extending the timeout is only a band-aid solution. It should not take up to 30s to enter a vertical. 

Personally I think the wait for networkidle may be a little overkill, but there must be a reason for the use of this wait. I suspect that without the wait the page/step "swallows" the actions Playwright ended up performing on the select field.